src/harness/testcases/WINNF_FT_S_FAD_testcase.py

```diff
@@ -398,7 +406,7 @@ def test_WINNF_FT_S_FAD_1(self, config_filename):
               ppa_dump_data.extend(downloaded_file['recordData'])
           else:
               self.assertEqual('coordination', dump_file['recordType'])
-          hash_of_dump_file.append({'url':dump_file['url'], 'hashValue' : hashlib.sha1(downloaded_file).hexdigest()} if dump_file is not None else None)
+          hash_of_dump_file[dump_file['url']] =  hashlib.sha1(downloaded_file).hexdigest() if dump_file is not None else None
```

Comment:
In what situations will `dump_file` be `None`? If it can happen here, then it can happen below (line 466) so your hash generation methodology should be the same in both places.